.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -100,25 +101,35 @@ Bugs & improvements:
 
 _____
 
-## OEM Release
+## Enterprise release
 
 ### TASKS:
 
-- [ ] [GIT] Create a new branch `release/M.m.p_oem` (optional)
-- [ ] [DIS] Update release notes in app with the proper content for oem release
-- [ ] [GIT] Create and sign tag `vM.m.p_oem` in HEAD commit of `release/M.m.p_oem` branch
+- [ ] [GIT] Create branch `release/M.m.p_enterprise` in owncloud/android from `latest` (or the corresponding release tag)
+- [ ] [DOC] Update [SBOM](https://infinite.owncloud.com/f/31e6d44f-f373-557c-9ab3-1748fc0c650d$4994cd9c-1c17-4254-829a-f5ef6e1ff7e3%215080be84-fbcc-4aca-956e-b278a7090418)
+- [ ] [DIS] Update release notes in app and changelog in `M.m.p_YYYY-MM-DD` (already released version) with the proper content for the release
+- [ ] [DIS] Copy the `unreleased` folder in `master` branch into this branch, to avoid Calens conflicts problems
+- [ ] [DEV] Check and reorder release notes in `ReleaseNotesViewModel.kt` to assure nothing important is missing there
+- [ ] [DEV] Code review
+- [ ] [DIS] Generate final bundle and APK from last commit in the release branch
+- [ ] [QA] Design test plan
+- [ ] [QA] Test execution
+- [ ] [QA] Trigger BitRise builds for unit tests and UI tests, in case changelog conflicts avoid them in GitHub
+- [ ] [QA] QA approval
+- [ ] [DIS] Upload release APK and bundle to internal ownCloud instance
+- [ ] [GIT] Create and sign tag `vM.m.p_enterprise` in HEAD commit of release branch, in owncloud/android
 - [ ] [DEV] Approve and merge changes in ownBrander
   - [ ] Feature 1 oB https://github.com/owncloud/ownbrander/pull/
   - [ ] Feature 2 oB https://github.com/owncloud/ownbrander/pull/
   - [ ] Update version number in ownBrander
-- [ ] [OPS] Block oB button
-- [ ] [OPS] Deploy oB
-- [ ] [QA] Generate final APKs files from signed commit in builder machine and perform some basic checks:
-    - [ ] Installation of apk/aab generated by builder machine
+- [ ] [COM] Block oB button
+- [ ] [COM] Deploy oB
+- [ ] [QA] Generate final APKs from signed commit in builder machine and perform some basic checks
+    - [ ] Installation of APK/aab generated by builder machine
```

Comment:
APK in capitals and aab not? any reason? In previous steps and definitions, you called `bundle` the `aab`. We can keep that naming across the whole file.

Reply:
I was aware that APK stands for "Android package", but didn't realise that AAB stands for "Android App Bundle" 😅. In any case, yeah, let's keep consistency and call it always "bundle" 👍